x-pack/test/api_integration/deployment_agnostic/services/slo_api.ts

```diff
@@ -95,6 +128,22 @@ export function SloApiProvider({ getService }: DeploymentAgnosticFtrProviderCont
       return body;
     },
 
+    async updateSavedObject(
+      roleAuthc: RoleCredentials,
+      slo: (UpdateSLOInput & { version: number }) | SavedObject<SLODefinitionResponse>,
```

Comment:
We have a StoredSLO type which represents the stored data, in the models/ 
```suggestion
      slo: StoredSLO,
```

Reply:
Where is this type defined? It's not appearing in search results.